cypress/e2e/tests/migration/migration-waves/field_validation.test.ts

```diff
@@ -60,19 +68,54 @@ describe(["@tier1"], "Migration Waves Validations", () => {
         const tomorrow = new RegExp("^" + (now.getDate() + 1) + "$");
         // Start date should be greater than actual date
         cy.contains("button", new RegExp("^" + now.getDate() + "$")).should("be.disabled");
-        cy.contains("button", tomorrow).should("be.enabled").click();
+        //making sure we are selecting correct element and not from last month as it is visible in the calendar
+        cy.get("td:not(.pf-m-adjacent-month)")
+            .contains("button.pf-c-calendar-month__date", tomorrow)
+            .should("be.enabled")
+            .click();
 
         cy.get(MigrationWaveView.endDateInput).next("button").click();
         // End date should be greater than start date
         cy.contains("button", tomorrow).should("be.disabled");
-        cy.contains("button", new RegExp("^" + (now.getDate() + 2) + "$"))
+
+        cy.get("td:not(.pf-m-adjacent-month)")
+            .contains(
+                "button.pf-c-calendar-month__date",
+                new RegExp("^" + (now.getDate() + 2) + "$")
+            )
             .should("be.enabled")
             .click();
 
         cy.get(MigrationWaveView.submitButton).should("be.enabled");
         cy.get(cancelButton).click();
     });
 
+    // Automates validations for Polarion TC 365
+    it("Unique validations", function () {
+        let migrationWavesList: MigrationWave[] = [];
+
+        let name1 = data.getRandomWord(8);
```

Comment:
Why is this called `name1` if there is no more `name` variables? It can be called just `name`.

It can also be a `const`